### PR TITLE
Group B chrisDong.md

### PR DESCRIPTION
**Pull request by MajesticUnic0rn**
please accept my pull request

### CODE REVIEW

File: _posts/2020-06-05-ChrisDongB.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: getting out of this covid quarantine 
+author: chris
+date: 2020-06-05
+---
+
+I just want to go outside please
```
Comment (vmjoseph) [inline _posts/2020-06-05-ChrisDongB.md:7]: We might not be able to go outside, but you win the award for best github username

File: _posts/2020-06-05-ChrisDongB.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: getting out of this covid quarantine
```
Comment (asingh9) [inline _posts/2020-06-05-ChrisDongB.md]:
```suggestion
title: This summer I'm looking forward to...
```

File: _posts/2020-06-05-ChrisDongB.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: getting out of this covid quarantine 
+author: chris
+date: 2020-06-05
+---
+
```
Comment (asingh9) [inline _posts/2020-06-05-ChrisDongB.md]:
```suggestion
getting out of this covid quarantine
```

**Review by MajesticUnic0rn — COMMENTED** [review]
.

**Pull request merged**